githubservice/githubservice.go

```diff
@@ -199,6 +199,7 @@ func (g *GithubService) loadCommitsFromAllRepoPRs(owner string, repo string, day
 	}
 
 	remainingPRsAreOlder := false
+	daysAsHours := float64(days) * 24
```

Comment:
We'll need to do this calculation one calling level up so we can pass the same time into this function as well:

func (g *GithubService) loadCommitsForRepo(owner string, repo string, committer string, days int) ([]github.RepositoryCommit, error) {

Or else there can be a slight drift in comparison times due to processing times over all the repositories...